mmv1/products/backupdr/BackupVault.yaml

```diff
@@ -0,0 +1,147 @@
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/backupVaults
+create_url: projects/{{project}}/locations/{{location}}/backupVaults?backupVaultId={{backup_vault_id}}
+update_url: projects/{{project}}/locations/{{location}}/backupVaults/{{backup_vault_id}}?force={{force_update}}
+delete_url: projects/{{project}}/locations/{{location}}/backupVaults/{{backup_vault_id}}?force={{force_delete}}&allowMissing={{allow_missing}}
+self_link: projects/{{project}}/locations/{{location}}/backupVaults/{{backup_vault_id}}
+id_format: projects/{{project}}/locations/{{location}}/backupVaults/{{backup_vault_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/backupVaults/{{backup_vault_id}}
+name: BackupVault
+description: Container to store and organize immutable backups.
```

Comment:
Are the backups immutable?

Reply:
Backup vault ensures that backups are both immutable & indelible. Reference: https://docs.google.com/presentation/d/1zFDIQh6DHciMbETvmi0ynt2WXtS4ujGL7y0fsOHnWxs/edit?resourcekey=0-w48LoHQeVwRZJrRL9WbN-Q#slide=id.g2761e27a707_0_558